repos/c2sm/packages/icon/package.py

```diff
@@ -588,6 +588,9 @@ def configure_args(self):
                 self.spec['py-icon4py:atm_dyn_iconam'].headers.directories[0])
             config_vars['LOC_ICON4PY_TOOLS'].append(
                 self.spec['py-icon4py:tools'].headers.directories[0])
+            config_vars['LOC_ICON4PY_DIFFUSION'].append(self.spec['py-icon4py:diffusion'].headers.directories[0])
```

Comment:
This breaks for older versions of icon4py.
Please add `if self.spec['py-icon4py'].version => ver(xxx)`.